Server/bkr/server/tools/beakerd.py

```diff
@@ -597,6 +597,8 @@ def provision_scheduled_recipeset(recipeset_id):
             recipe.waiting()
             recipe.provision()
         except Exception as e:
+            # Make sure that rollback is first instruction here before touching ORM again
+            # Otherwise, ORM will raise another exception
```

Comment:
Maybe tell them that a log message containing a recipe.id "touches the ORM"

Reply:
I will leave it open. It is possible that the debug message will change. You have to be aware when you working with ORM attributes.